clients/cloud/ksql-datagen/README.md

```diff
@@ -14,10 +14,10 @@ To run this example, create a local file with configuration parameters to connec
 If this is a Confluent Cloud cluster, you must have:
 
 * Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) cluster
-* Initialize a properties file at `$HOME/.ccloud/config` with configuration to your Confluent Cloud cluster:
+* Initialize a properties file at `$HOME/.confluent/config` with configuration to your Confluent Cloud cluster:
```

Comment:
Same -- needs to replaced & deleted (here and other places), like the clojure README